help/md/Integer.md

```diff
@@ -1,9 +1,17 @@
 ## name
 Integer
 ## title
+Integer
 ## description
+Integer Datatype
 ## details
+This datatype includes all whole, positive and negative numbers.
 ## authors
 ## see_also
+Natural
+IntegerPos
+RealPos
```

Comment:
Maybe remove RealPos from this?  If you are going to mention real numbers, it seems that `Real` would be a better thing to reference, since Integer can be negative.